### ISSUE TITLE
FTX -> Binance US 

### ISSUE THREAD

**Issue by zkbitcoin** [sc1f/perspective-btc-liquidity#1]
first, excellent art work @ https://juntan.me/Selected-Work really mean it , stands out

for here, what would be a smooth transition to showcase this project now that FTX stream API's are done for (next logical would be binance US for BTC/USDT pair) 

tried naive switch to const BINANCE_URL = "wss://stream.binance.us:9443/ws/btcusdt@bookTicker";

but it led to further errors , worry that something is embedded in perspective itself tying into FTX somewhere

**Comment by sc1f** [comment]
yeah, "FTX collapse" definitely was not on my bingo card when I spun this up

Perspective itself are about as data-agnostic as it gets; just switching the URL however won't work because the format of the data is different. Inside [datasource.tsx](https://github.com/sc1f/perspective-btc-liquidity/blob/master/src/datasource.tsx#L158) you can see the `DataSource` interface, as well as a stubbed out Binance impl that I never finished.

I will give switching over a go now that FTX is dead - thanks for the flag!

**Comment by BozidarSiljanoski** [comment]
sorry for ignorant question, but would binance US be different than the other binance?

**Comment by zkbitcoin** [comment]
> sorry for ignorant question, but would binance US be different than the other binance?

from my experience binance.com (non US) and binance.us appear to be exactly the same just asset selection differs (both wss and http access)
